blocks/research/__inner-wrapper/research__inner-wrapper.css

```diff
@@ -0,0 +1,4 @@
+.research__inner-wrapper {
```

Comment:
@Woolfs1423 Now that we've aligned the elements within `research__inner-wrapper` in relation to each other, let's also center them inside their container by applying the CSS property `justify-content: center;` for `research__inner-wrapper`.